omod-2.4/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_4/PatientController2_4Test.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.controller.openmrs2_4;
+
+import org.junit.Ignore;
+import org.junit.Test;
+import org.openmrs.Patient;
+import org.openmrs.PatientIdentifier;
+import org.openmrs.Person;
+import org.openmrs.PersonName;
+import org.openmrs.api.context.Context;
+import org.openmrs.api.db.PatientDAO;
+import org.openmrs.api.db.PersonDAO;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.test.Util;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.RestControllerTestUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.sql.Date;
+
+import static org.junit.Assert.assertEquals;
+
+public class PatientController2_4Test extends RestControllerTestUtils {
+
+	@Autowired
+	private PatientDAO hibernatePatientDao;
+
+	@Autowired
+	private PersonDAO hibernatePersonDAO;
+
+	@Test
+	@Ignore("TRUNK-6010: The generated SQL contains syntax error in Core before version 2.5. This has been fixed on commit c3f63301. Once a submodule that depends on core version 2.5 is added, this test should be moved there and unignored.")
+	public void shouldReturnDuplicatesPatients() throws Exception {
```

Comment:
Can we we fix the grammar in the method name? 😊

Reply:
Of course :) https://github.com/openmrs/openmrs-module-webservices.rest/pull/488/commits/b0ef54f9fa6569c3e7af83d7d49ff69d23886af6